src/Stratis.Bitcoin.Features.ColdStaking/ColdStakingDestinationReader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+using Stratis.Bitcoin.Interfaces;
+
+namespace Stratis.Bitcoin.Features.ColdStaking
+{
+    public class ColdStakingDestinationReader : ScriptDestinationReader, IScriptDestinationReader
```

Comment:
Can we add a comment on why ColdStaking requires its own `ScriptDestinationReader`?

Reply:
Yes. The base `ScriptDestinationReader` was introduced with this PR to offer the ability to selectively replace `ScriptAddressReader` which can only parse a single address from a `ScriptPubKey`.  ColdStaking scripts contain two addresses / pub key hashes.

Reply:
Looks good, please add to the class 👍